src/cf.data.pre

```diff
@@ -3338,8 +3349,13 @@ IFDEF: USE_OPENSSL
 DEFAULT: 32 startup=5 idle=1 concurrency=1
 LOC: Ssl::TheConfig.ssl_crt_validator_Children
 DOC_START
-	The maximum number of processes spawn to service SSL server.
-	The maximum this may be safely set to is 32.
+	Specifies the maximum number of certificate validation processes that
+	Squid may spawn (numberofchildren) and several related options. Using
+	too few helper processes queues requests. Using too many helpers
```

Comment:
It should be called either "helper processes" or just "helpers".

Reply:
Please see if you like the rephrased version better.